blueprints/blueprint-test/files/__root__/__name__-test.js

```diff
@@ -1,21 +1,24 @@
 'use strict';
 
-const blueprintHelpers = require('ember-cli-blueprint-test-helpers/helpers');
-const setupTestHooks = blueprintHelpers.setupTestHooks;
-const emberNew = blueprintHelpers.emberNew;
-const emberGenerateDestroy = blueprintHelpers.emberGenerateDestroy;
+const { 
+  setupTestHooks,
+  emberNew,
+  emberGenerateDestroy,
+} = require('ember-cli-blueprint-test-helpers/helpers');
 
-const expect = require('ember-cli-blueprint-test-helpers/chai').expect;
+const {
+  expect
+} = require('ember-cli-blueprint-test-helpers/chai');
 
 describe('Acceptance: ember generate and destroy <%= blueprintName %>', function() {
   setupTestHooks(this);
 
   it('<%= blueprintName %> foo', function() {
-    let args = ['<%= blueprintName %>', 'foo'];
+    const args = ['<%= blueprintName %>', 'foo'];
 
     // pass any additional command line options in the arguments array
     return emberNew(<%= muOption %>)
-      .then(() => emberGenerateDestroy(args, (file) => {
+      .then(() => emberGenerateDestroy(args, () => {
```

Comment:
To remove the mystery of where `file` comes from, maybe we should doe `(/* file */)`?

Reply:
👍